mlir/lib/Dialect/Tensor/Transforms/PackAndUnpackPatterns.cpp

```diff
@@ -11,6 +11,7 @@
 #include "mlir/Dialect/Tensor/Transforms/Transforms.h"
 #include "mlir/Dialect/Utils/IndexingUtils.h"
 #include "mlir/IR/PatternMatch.h"
+#include "mlir/Support/LogicalResult.h"
```

Comment:
Is this include needed?

Reply:
It defines LogicalResult, but we usually dont include it. Let me remove it.